src/core/runtime/runtime.cc

```diff
@@ -84,6 +86,7 @@ static const char* const core_library_name = "legate.core";
   parse_variable("LEGATE_EMPTY_TASK", use_empty_task);
   parse_variable("LEGATE_SYNC_STREAM_VIEW", synchronize_stream_view);
   parse_variable("LEGATE_LOG_MAPPING", log_mapping_decisions);
+  parse_variable("LEGATE_WARMUP_NCCL", warmup_nccl);
```

Comment:
I just realized these environment variables are not documented anywhere. In a follow-up PR we should add them to the listing in https://github.com/nv-legate/legate.core/blob/branch-23.09/legate/settings.py.